sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md

```diff
@@ -1,11 +1,10 @@
 # Release History
 
-## 1.0.0b3 (Unreleased)
-
-## 1.0.0b2 (Unreleased)
+## 1.0.0b2 (2023-06-12)
```

Comment:
When do you actually plan to release this?

Reply:
Changed release date to Monday. Will create an out-of-band release then.